docs/spec/spec.md

```diff
@@ -0,0 +1,310 @@
+| Update | The Provisioner will synchronize the Channel backing resources to reflect the update.                                                                            |                                                                      |
+| Delete | The Provisioner will deprovision the backing resources if no longer required depending on implementation.                                                        |                                                                      |
+
+---
+
+## kind: Provisioner
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes an abstract configuration of a Source system which produces events
+or a Channel system that receives and delivers events._
+
+### Object Schema
+
+#### Spec
+
+| Field  | Type                  | Description                                 | Limitations                |
+| ------ | --------------------- | ------------------------------------------- | -------------------------- |
+| type\* | GroupKind<sup>1</sup> | The type of the resource to be provisioned. | Must be Source or Channel. |
+
+\*: Required
+
+1: Kubernetes type.
+
+#### Metadata
+
+##### Owner References
+
+- Owns EventTypes.
+
+#### Status
+
+| Field       | Type                      | Description                                                          | Limitations                                                                    |
+| ----------- | ------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| provisioned | []ProvisionedObjectStatus | Status of creation or adoption of each EventType and errors therein. | It is expected that a provisioner list all produced EventTypes, if applicable. |
+
+#### Events
+
+- Source created
+- Source deleted
+- Event types installed
+
+### Life Cycle
+
+| Action | Reactions                                                                       | Limitations                                                                                              |
+| ------ | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Create | Creates and owns EventTypes produced, or adds Owner ref to existing EventTypes. | Verifies Json Schema provided by existing EventTypes; Not allowed to edit EventType if previously Owned; |
+| Update | Synchronizes EventTypes.                                                        |                                                                                                          |
+| Delete | Removes Owner ref from EventTypes.                                              |                                                                                                          |
+
+---
+
+## kind: Subscription
```

Comment:
I'd hoist Subscription over Provisioner, unless you're doing this purely alphabetically.